home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/exception/PasswordException.java

```diff
@@ -0,0 +1,7 @@
+package com.softserveinc.ita.homeproject.homeservice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_ACCEPTABLE, reason = "Password too weak")
```

Comment:
Please, don’t use default error message for the whole exception class. Due to further scale ability we could have a scenario where we’ll should use this class with custom error message

Reply:
Did you mean scalability?

Reply:
Removed reason. Updated to extend `BaseHomeException` which allows custom error message now.